.github/workflows/build_and_deploy.yml

```
@@ -69,4 +74,34 @@ jobs:
         uses: actions/upload-artifact@v3
         with:
           name: docs
-          path: docs/docs/_build
+          path: docs/docs/_build/html
+
+  deploy:
+    if: github.event_name == 'push' && (startsWith(github.ref, 'refs/tags/v') || startsWith(github.ref, 'refs/heads/main'))
+    name: Build & Upload Artifact
+    runs-on: ubuntu-latest
+    steps:
+      - name: Download artifact
+        uses: actions/download-artifact@v2
+        with:
+          name: docs
+          path: docs
+
+      - name: get directory name
+        # if this is a tag, use the tag name as the directory name else dev
+        run: |
+          if [[ "${{ github.ref }}" == "refs/tags/"* ]]; then
+            echo "branch_name=${{ github.ref/refs\/tags\// }}" >> $GITHUB_ENV 
```

```suggestion
            echo "branch_name=${{ github.ref\/refs\/tags\// }}" >> $GITHUB_ENV 
```
I think this fixes the error:
https://github.com/napari/docs/actions/runs/8103613500